src/controllers/userController.js

```diff
@@ -26,7 +38,7 @@ const getSingleUser = async (req, res) => {
 				schema: { $ref: '#/definitions/PasswordlessUser' }
```

Comment:
I missed in this example one that this endpoint returns `{ user: < passwordless user> }` so this should be updated to:

```suggestion
				schema: { user: { $ref: '#/definitions/PasswordlessUser' } }
```